neurodegeneration/processing-containers/normative_biomarker_visualizer/files/normative_biomarker_visualizer.py

```diff
@@ -205,26 +205,34 @@ def plotWithRef(dfRef, WMLSref, dfSub, fname, spareAD, spareBA):
 		setmin = setmin - spacer
 
 		if selVar == 'Total White Matter Hyperintensity Volume':
-			CN_up5 = [abs(int(i)-(int(dfSub['Age'].values[0])+3)) for i in XX_CN]
-			CN_down5 = [abs(int(i)-(int(dfSub['Age'].values[0])-3)) for i in XX_CN]
+			if dfSub[selVar].tolist()[0] != 0:
```

Comment:
@dwu12  Can you please add a comment stating that this if block is executed when wmls output is found? 
Makes it easy to read the code.